### ISSUE TITLE
Question on Aerial BEV sampling

### ISSUE THREAD

**Issue by SummerpanKing** [vita-epfl/FG2#3]
Thank you for open-sourcing this impressive and inspiring work! I have a question regarding the aerial BEV sampling implementation in the code.
In particular, on line 172 of models/model_vigor.py, the satellite feature is resized via:
---
sat_feature_bev = nn.functional.interpolate(sat_feature, (self.sat_bev_res, self.sat_bev_res), mode='bilinear')
---
However, as mentioned in the paper, the satellite-to-BEV projection is expected to incorporate ground sampling distance (GSD). This interpolation seems to perform purely resolution-based resampling, potentially ignoring the role of GSD.

Based on our understanding, GSD-based sampling is important for aligning satellite BEV features with ground BEV queries under a 3-DoF constraint, which in turn is crucial for generating pseudo-labels to supervise the network.
That said, we might have misunderstood this part, and we would truly appreciate any clarification from you.

Best regards.

**Comment by SummerpanKing** [comment]
Ah, I see now!
The GSD consideration is implicitly embedded in the construction of the BEV queries—what a clever and elegant design!
